pkg/testutil/testutil.go

```diff
@@ -0,0 +1,36 @@
+// Copyright 2016 PingCAP, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package testutil
+
+import (
+	"net"
+
+	. "github.com/pingcap/check"
+	"github.com/pingcap/kvproto/pkg/pdpb"
+	"github.com/pingcap/pd/pkg/rpcutil"
+)
+
+func MustRPCCall(c *C, conn net.Conn, request *pdpb.Request) *pdpb.Response {
```

Comment:
I think it's not very meaningful to add comment for these test wrappers, and I have filtered out `testutil` from `golint`.